__tests__/api/api.test.js

```diff
@@ -165,6 +165,79 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("FEATURE: GET /api/articles (queries)", () => {
+  test("Articles should be filtered by topic=mitch, sorted by author and ordered ascending ", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch&sort_by=author&order=asc")
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("author", { descending: false });
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("topic", "mitch");
+        });
+        expect(articles[0]).toStrictEqual({
```

Comment:
You could check all articles here, also using a more flexible matcher like .toMatchObject() or .objectContaining() to allow future expansion of articles.